features/create_membership_application.feature

```diff
@@ -227,6 +231,27 @@ Feature: Create a new membership application
     And I should see t("shf_applications.create.success_with_app_files_missing")
 
 
+  @selenium
+  Scenario: A user can submit a new Membership Application without sub categories
+    Given I am on the "user instructions" page
+    And I click on first t("menus.nav.users.apply_for_membership") link
+    And I fill in the translated form with data:
+      | shf_applications.show.company_number | shf_applications.new.phone_number | shf_applications.new.contact_email |
+      | 5560360793                           | 031-1234567                       | info@craft.se                      |
+
+    And I should not see "overall grooming" Category
+      | 5560360793                           | 031-1234567                       | info@craft.se                      |
+
+    And I should not see "overall grooming"
```

Comment:
Checking that the subcategory is not present in the form - when the user creates or edits an SHF application - is good.  The problem is that we don't explicitly know if that subcategory is actually present in the DB, or the user does not see it because it is _not_ in the DB.  To fix that, I suggest having the admin log in before the user, and have the admin go to the business categories page to confirm that the subcategory is actually present - before the user logs in to create or edit the application. (see examples of that in `features/business_categories/edit_business_category.feature`).